tests/VolleyManagement.UnitTests/Mvc/ViewModels/TeamNameViewModelComparer.cs

```diff
@@ -61,6 +61,24 @@ public bool AreEqual(TeamNameViewModel x, TeamNameViewModel y)
             Assert.AreEqual(x.Id, y.Id, "Id should be equal");
             Assert.AreEqual(x.Name, y.Name, "Name should be equal");
             Assert.AreEqual(x.DivisionName, y.DivisionName, "DivisionName should be equal");
+            Assert.AreEqual(x.GroupName, y.GroupName, "GroupName should be equal");
+
+            return true;
+        }
+
+        /// <summary>
+        /// Finds out whether two lists of team objects have the same properties.
+        /// </summary>
+        /// <param name="x">The first list of object to compare.</param>
+        /// <param name="y">The second list of object to compare.</param>
+        /// <returns>True if given lists of team have the same properties.</returns>
+        public bool AreEqual(List<TeamNameViewModel> x, List<TeamNameViewModel> y)
+        {
+            for (int i = 0; i < x.Count; i++)
```

Comment:
What if x is null?

Reply:
Will you be able to tell why it crashed?

Reply:
@sdiachen Ok, you are right, I understood

Reply:
@sdiachen Sure:
![image](https://user-images.githubusercontent.com/36235893/36791403-63f291f8-1ca0-11e8-8e71-3097f1451ecf.png)

![image](https://user-images.githubusercontent.com/36235893/36791449-8b3b838c-1ca0-11e8-8074-c53f73dd549e.png)

Reply:
What if test runs on CI?